index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;300;400;600&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Document</title>
+  </head>
+
+  <body>
+    <div class="menu-bar">
+      <div class="menu">
+        <div id="bar1" class="bar"></div>
+        <div id="bar2" class="bar"></div>
+        <div id="bar3" class="bar"></div>
```

Comment:
- [ ] Kindly ensure that your hamburger matches the Figma design you chose 👍🏽 . Kindly adjust the font-weight and color of your hamburger.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/191672823-f5cdc7a9-495c-4e0a-8bad-89f08b8ee861.png))  | ![image](https://user-images.githubusercontent.com/63473584/191672743-90683e4d-ed5a-4982-ab14-7c1b49bde3f5.png)  |